instrumentation/log4j/log4j-2.13.2/library/build.gradle.kts

```diff
@@ -3,11 +3,16 @@ plugins {
 }
 
 dependencies {
+  compileOnly("io.opentelemetry:opentelemetry-sdk-logs:1.9.0-alpha")
```

Comment:
The repo's updated to 1.9 so should be able to merge main and remove the versions now